source/ShipManager.cpp

```diff
@@ -0,0 +1,189 @@
+/* ShipManager.cpp
+Copyright (c) 2022 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "ShipManager.h"
+
+#include "DataNode.h"
+#include "DataWriter.h"
+#include "EsUuid.h"
+#include "GameData.h"
+#include "Messages.h"
+#include "PlayerInfo.h"
+#include "Ship.h"
+
+#include <cstdlib>
+
+using namespace std;
+
+
+
+void ShipManager::Load(const DataNode &node)
+{
+	if(node.Size() < 3 || node.Token(1) != "ship")
+	{
+		node.PrintTrace("Error: Skipping unsupported \"" + token + "\" syntax:");
```

Comment:
```suggestion
		node.PrintTrace("Error: Skipping unrecognized node.");
```

Reply:
I wish I had seen that bruh this is a better way to say it